index.html

```diff
@@ -1414,8 +1447,43 @@ <h3>Selective Disclosure</h3>
       <section>
         <h3>Predicates</h3>
         <p>
-Predicates allow a <a>holder</a> to provide True/False values to a
-<a>verifier</a> rather than revealing claim values.
+Another technique which may be used to support progressive trust is to use
+predicates as the values of revealed claims. Predicates allow a <a>holder</a> to
+provide True/False values to a <a>verifier</a> rather than revealing claim
+values.
+        </p>
+        <p>
+Predicate proofs may be enabled by <a>verifiable credential</a> <a>issuers</a>
+as claims, e.g., the <a>credentialSubject</a> may include a
+<code>ageOver18</code> <a>property</a> rather than a <code>birthdate</code>
+<a>property</a>. This would allow <a>holders</a> to provide proofs that they are
```

Comment:
`proofs` 
-> `proofs`

Reply:
I'm not sure what change you're proposing here.

Reply:
My suggestion is 
`This would allow <a>holders</a> to provide proofs that`
-> `This would allow <a>holders</a> to provide proof that`
because most people don't use the plural for this case, and it reads awkwardly to me.

Reply:
Fixed in [4da4edd](https://github.com/w3c/vc-imp-guide/pull/14/commits/4da4edd1dd2bf0b6c136afa635246dcc8caf20f5)